### DISCUSSION TITLE
What means the key `page_id`  ?

### DISCUSSION THREAD

**Discussion by JulioAzcarate** [george-gca/multi-language-al-folio#26] [Q&A] [answered]
Hi George

I would like to understand why do you defined the key `page_id` in the font-matter of some documents (eg. in projects) or pages ?

Thanks in advances!
Julio

**Comment by george-gca** [comment, marked as answer]
This is a key used by the [polyglot](https://github.com/untra/polyglot) plug-in to help identify pages. It is useful, for example, when you have a page in a different language with a different permalink.
